content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Nomisr - Access 'Nomis' UK Labour Market Data"
+date: 2018-04-27
+package_version: 0.2.0
+authors:
+  - name: Evan Odell
+    url: https://github.com/evanodell
+categories:
+  - technotes
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- data-access
+- data-extraction
+- geospatial
+- uk
+- census
+- geography
+---
+
+```{r echo=FALSE}
+knitr::opts_chunk$set(
+  comment = "#>",
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE
+)
+```
+
+
+I'm excited to announce a new package for accessing official statistics from the UK. [`nomisr`](https://github.com/ropensci/nomisr) is the R client for the [Nomis](https://www.nomisweb.co.uk/) database. Nomis is run by Durham University on behalf of the UK's Office for National Statistics (ONS), and contains over a thousand datasets, primarily on the UK labour market, census data, benefit spending and general economic activity. Registration is optional, although registration and the use of an API key allows for larger queries without the risk of being timed out or rate limited by the API.
+
+Most data in Nomis is arranged in statistical geographies -- ranging from the whole of the UK to small output areas, making `nomisr` a useful tool for map-making and social geography.
+
+## Installation
+
+
+`nomisr` is on [CRAN](https://cran.r-project.org/package=nomisr), so it can be installed with:
+```{r eval=FALSE}
+install.packages("nomisr")
+```
+
+As of writing, the development and CRAN versions are the same, but the development version can still be installed with
+```{r eval=FALSE}
+remotes::install_github("ropensci/nomisr")
+```
+
+## Using `nomisr`
+
+The [vignette](https://cran.rstudio.com/web/packages/nomisr/vignettes/introduction.html) has more details on identifying available datasets and the parameters they accept, but the example below shows the latest data on Job Seekers Allowance claimants in each parliamentary constitutency. 
+
+
+```{r}
+library(nomisr)
+
+jsa_constits <- nomis_get_data(id = "NM_1_1", time = "latest", 
+                               measures = 20100, sex = 7,
+                               geography = "TYPE460", 
+                               additional_queries = "&item=1")
+```
+
+
+
+## To do
+
+The `nomisr` package covers most of the features available on Nomis, with the exception of a grouping function that can easily be replicated in R, and that will likely be incorporated into a future release. Long term development is focused on making the package more user-friendly for people with less familiarity with Nomis data and the API structure.
+
+### Package review
+
+Thank you to reviewers [Paul Egeler](https://github.com/pegeler) and [Christophe Dervieux](https://github.com/cderv) for their incredibly helpful comments and suggestions, and for [Karthik Ram](https://github.com/karthik)'s editorial supervision. `nomisr` is a much better package than would otherwise have been the case as a result of their efforts.
```

Comment:
consider adding link to the open review thread: https://github.com/ropensci/onboarding/issues/190